frontend/src/components/shared/answersetView/StoredAnswersetViewer.jsx

```diff
@@ -35,8 +35,28 @@ export default function StoredAnswersetView({ question_id, answer_id }) {
       return;
     }
 
-    const questionResponseJSON = JSON.parse(questionResponse);
-    const answerResponseJSON = JSON.parse(answerResponse);
+    let questionResponseJSON;
+    try {
+      questionResponseJSON = JSON.parse(questionResponse);
+    } catch (err) {
+      pageStatus.setFailure('Invalid question JSON');
+      return;
+    }
+
+    let answerResponseJSON;
+    try {
+      answerResponseJSON = JSON.parse(answerResponse);
+    } catch (err) {
+      pageStatus.setFailure('Invalid answer JSON');
+      return;
+    }
+
+    if (answerResponseJSON.status === 'error') {
```

Comment:
Why don't we do this for the question response?

Reply:
The question will never have a status or message property. There is no way to upload an invalid question to robokache through the UI.

Reply:
Not even if it was uploaded from the simple viewer? Or if someone uploads directly to Robokache? Or somehow the file is deleted but the database isn't updated? If you're confident that there's no possible way for anything bad to happen, then that's fine. But it's also just 4 lines of code.

Reply:
It's not that there's no possibility of something bad happening. There is for sure a chance that the question data won't be a valid JSON object or that it will be unparseable. However, it's impossible in the current setup of the query-dispatcher to create a question that has a `status === 'error'` property. Invalid questions are rejected and if we did get an invalid question uploaded it would fail at either the JSON parse or the message parse below it.

Reply:
Ok